### ISSUE TITLE
GB Cable Link Transfer - Wrong Password

### ISSUE THREAD

**Issue by dezign999** [EstebanFuentealba/MALVEKE-Flipper-Zero#5]
Having trouble launching the app, I cannot get it to get passed "connect malveke" screen unless I press the RST button and immediately the OK button. Once in, I enter 12345678 as the password, but it always comes back as incorrect on my Mac and iPhone.

**Comment by EstebanFuentealba** [comment]
Hi!, Do you mean connecting MALVEKE with the GAMEBOY and the GAME BOY Camera? For that, you don't need to press the RST.

You need to follow these usage **[Instructions for use](https://github.com/EstebanFuentealba/MALVEKE-Flipper-Zero/blob/main/flipper_companion_apps/applications/external/malveke_gb_link_camera/README.md#instructions-for-use)** step by step and ensure that when connecting the link cable (**After being connected to the MALVEKE WIFI network**), it's in the `EXT2` slot (the bottom port).


![GAME-BOY-MALVEKE-connection-EXT2 (1)](https://github.com/EstebanFuentealba/MALVEKE-Flipper-Zero/assets/442927/7b6ce0bb-b84a-42d9-84fb-ea3a19612e19)

**Comment by dezign999** [comment]
Hi, I am doing exactly what the instructions state, however it never goes into hotspot mode by itself. Pressing the OK button just makes the red light on the Malveke blink. I can force the hotspot mode by pressing the Reset button followed by the OK button. The hotspot will then show up on the list of available hotspots. When connecting, the default password says it’s incorrect. I have flashed the malveke firmware twice to ensure it was correct and still the same result. Not sure if this is a hardware issue or software issue. 

I am using the default flipper firmware, current version, and the faps from labs.flipper site.

**Comment by EstebanFuentealba** [comment]
I didn't have any boards left to test, so I put one together (it doesn't have LEDs or ports), but I recorded the test. I did it using a GAME BOY Pocket, but I could try with others.

https://youtu.be/rWLYpNNkJOg

To rule out a few things:
1. Which GAME BOY do you have?
2. From which version of iOS/macOS did you connect?
3. Have you tried with any Android device?
4. Is Marauder working for you?

Best regards!

**Comment by dezign999** [comment]
Mine does not get to step 2. It will not go to the hotspot screen displaying the wifi credentials. I have to force getting that screen, so it may not be working correctly. I can make a video when I get home. It must be a hardware issue. I have not tried anything past logging in because I cannot connect. Current macOS, m1, iPhone 14 max pro, analogue pocket.

**Comment by EstebanFuentealba** [comment]
I understand. It's highly likely that you're right, and it might be a hardware issue. We have two options:

- Refund the order amount.
- Send you another board (shipping costs are on my part).

Let me know your decision, and please send me a message via Tindie indicating your username and order details.

Best regards!

**Comment by EstebanFuentealba** [comment]
Do you know what it could be? Perhaps there's a bit of flux residue causing some noise. If you have isopropyl alcohol, you can gently brush the cartridge pins and the ESP32 pins with an old toothbrush dipped in the alcohol.

**Comment by dezign999** [comment]
I tried cleaning the board with IPA, but I’m not certain it did anything, especially since Marauder works without any issues whatsoever. Now that Flipc.org is working again, I reflashed back to Unleashed firmware and the GB Cam link fap works much better now, nearly 95% success rate on launch. I’m curious is this is due to any variances in the esp32 hardware that the default flipper firmware hasn’t caught yet.
